tests/py_test.py

```diff
@@ -59,6 +59,57 @@ def assertRaises(test_case, fn, exception_class):
     test_case.assertTrue(exc is not None)
     test_case.assertTrue(isinstance(exc, exception_class))
 
+class AlignmentError(AssertionError):
```

Comment:
A new class that is only used in 2 place. Please inline and instead use the existing assertions.

Reply:
I have inlined this class.